src/api/MarkerBasedAPI.js

```diff
@@ -9,7 +9,7 @@ import type { ElementsErrorCallback } from '../common/types/api';
 
 type Params = {
     limit: number,
-    marker: string,
+    next_marker?: string,
```

Comment:
This base class is used by several different subclasses, so this would likely cause other issues. Is `marker` the typical convention for our other services? If so, it seems like the underlying service need to be updated to use that, instead?

Reply:
Ohh I see, hm..not sure if it's the typical convention for other services, but for annotations, it should be `next_marker` instead of `marker` (also, for annotations, the `next_marker` param should only be passed if there are annotations over the limit)

Reply:
I understand. Can we update the service to follow the [standard](https://developer.box.com/guides/api-calls/pagination/marker-based/), since pagination is not currently working and the endpoint is not yet public?